integration/list-boards.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
+var fs = require('fs')
+var path = require('path')
+
+var BOARDS = []
+var boardFolders = fs.readdirSync(path.join(__dirname, 'boards'))
+var count = boardFolders.length
+
+var done = function() {
+  if (--count === 0) {
+    fs.writeFileSync(path.join(__dirname, 'boards.json'), JSON.stringify(BOARDS, null, 2))
+  }
+}
+
+boardFolders.forEach(function(board) {
+  fs.readdir(path.join(__dirname, 'boards', board), function(board, error, files) {
+    if (error) {
+      throw error
+    }
+
+    var dir = path.join('boards', board)
+    var options = {}
+    var optionsExists = files.findIndex(function(file) {return file === 'options.json'}) >= 0
+
+    if (optionsExists) {
+      options = require(path.join(__dirname, dir, 'options.json'))
+    }
+
+    var layers = files.map(function(file) {
```

Comment:
This is tripping up for me because I happen to have OSX files in the directories with the gerbers (good 'ol .DS_Store). I'd consider putting the file list as an array in the options.json file itself. You could also use [glob](https://github.com/isaacs/node-glob) to get rid of the directory walk boilerplate. If you add the filelist to options.json, you can use a glob to grab all options.jsons in one go, or if you don't, glob won't match dotfiles so that would still help